build/common.ps1

```diff
@@ -83,15 +83,19 @@ function global:Get-MsBuildPath()
         }
     }
 
-    $path = & $VSWHERE_CMD -version "[15.0,16.0)" -requires Microsoft.Component.MSBuild -find "MSBuild\**\Bin\MSBuild.exe"
-    if (Test-Path $path)
+    $path = & $VSWHERE_CMD -version "[15.0,16.0)" -requires Microsoft.Component.MSBuild -find "MSBuild\**\Bin\MSBuild.exe" -latest | Select-Object -First 1
+    if ($path -and (Test-Path $path))

```

Comment:
Check `$path` is not null, first, sincel `Test-Path` will slightly obscure error "Cannot bind argument to parameter 'Path' because it is null."